codegen/method.go

```diff
@@ -162,6 +163,7 @@ func NewMethod(
 	var err error
 	var ok bool
 	method.Name = funcSpec.MethodName()
+	method.AdjustedName = CorrectMethodNaming(method.Name)
```

Comment:
mm ... what's the purpose for `adjustedName`?

Reply:
@cl1337 in `tchannel_client_test_server.tmpl` we was using method name to generate test server function type and struct type, which breaks lint since method name can be `testUuid` etc. Since method name is used in a lot of other places, I create the `AdjustedName` for holding initialism adjusted method name.